src/components/Modal/GatewayModal.js

```diff
@@ -115,192 +114,209 @@ const GatewayModal = ({ isOpen, onClose, onSubmit, CurrentNode, handleColorClick
-                          marginBottom: '20px',
-                          gap: '15px',
-                      }}
-                  >
-                      <div
-                          style={{
-                              width: '30px',
-                              height: '30px',
-                              borderRadius: '50%',
-                              backgroundColor: '#ffc9c9',
-                              cursor: 'pointer',
-                          }}
-                          onClick={() => handleColorClick('#ffc9c9')}
-                      ></div>
-                      <div
-                          style={{
-                              width: '30px',
-                              height: '30px',
-                              borderRadius: '50%',
-                              backgroundColor: '#b2f2bb',
-                              cursor: 'pointer',
-                          }}
-                          onClick={() => handleColorClick('#b2f2bb')}
-                      ></div>
-                      <div
-                          style={{
-                              width: '30px',
-                              height: '30px',
-                              borderRadius: '50%',
-                              backgroundColor: '#a5d8ff',
-                              cursor: 'pointer',
-                          }}
-                          onClick={() => handleColorClick('#a5d8ff')}
-                      ></div>
-                      <div
-                          style={{
-                              width: '30px',
-                              height: '30px',
-                              borderRadius: '50%',
-                              backgroundColor: '#ffec99',
-                              cursor: 'pointer',
-                          }}
-                          onClick={() => handleColorClick('#ffec99')}
-                      ></div>
-                      <div
-                          style={{
-                              width: '30px',
-                              height: '30px',
-                              border: '1px solid #cfcfcf',
-                              borderRadius: '50%',
-                              backgroundColor: '#fff',
-                              cursor: 'pointer',
-                          }}
-                          onClick={() => handleColorClick('#fff')}
-                      ></div>
-                  </div>
-                  <Button
-                      onClick={() => !duplicateApplicationNameError && onSubmit(ApplicationData)}
-                      style={{ display: 'block', margin: '0 auto' }}
-                      isDisabled={isSubmitDisabled || appNameCheck || serverPortCheck || portNumberError || portNumberRangeCheck}
-                  >
-                      Save
-                  </Button>
-              </ModalBody>
-          </ModalContent>
-      </Modal>
-  );
+    return (
```

Comment:
if there is any reformatting done on the code, it will be hard to understand what was the change made.
from next time  no reformatting....
but as reformatting the code is good practise. we will do it in coming day and every one will be in sync!